test/e2e/tests/tokens/increase-token-allowance.spec.js

```diff
@@ -283,7 +280,7 @@ describe('Increase Token Allowance', function () {
       text: `${finalSpendingCap} TST`,
       css: '.mm-box > h6',
     });
-    await driver.clickElement({
+    await driver.clickElementAndWaitForWindowToClose({
```

Comment:
This is the main change where the dialog needs to be closed too.

Reply:
Yay, you're using David's new function!

Reply:
yeah that's right!!